File: CS/DXSample/DataObjects/Resource.cs
using DevExpress.Xpo;
using System;
using System.Net;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Documents;
using System.Windows.Ink;
using System.Windows.Input;
using System.Windows.Media;
using System.Windows.Media.Animation;
using System.Windows.Shapes;

namespace DXSample.DataObjects {
    [Persistent("Resources")]
    public class Resources : XPLiteObject {
        public Resources(Session session) : base(session) { }

        [Key(true)]
        public int UniqueID { get; set; }
        public int ResourceId { get; set; }
        public string ResourceName { get; set; }
    }
}

File: VB/DXSample/DataObjects/Resource.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Xpo
Imports System
Imports System.Net
Imports System.Windows
Imports System.Windows.Controls
Imports System.Windows.Documents
Imports System.Windows.Ink
Imports System.Windows.Input
Imports System.Windows.Media
Imports System.Windows.Media.Animation
Imports System.Windows.Shapes

Namespace DXSample.DataObjects
	<Persistent("Resources")> _
	Public Class Resources
		Inherits XPLiteObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private privateUniqueID As Integer
		<Key(True)> _
		Public Property UniqueID() As Integer
			Get
				Return privateUniqueID
			End Get
			Set(ByVal value As Integer)
				privateUniqueID = value
			End Set
		End Property
		Private privateResourceId As Integer
		Public Property ResourceId() As Integer
			Get
				Return privateResourceId
			End Get
			Set(ByVal value As Integer)
				privateResourceId = value
			End Set
		End Property
		Private privateResourceName As String
		Public Property ResourceName() As String
			Get
				Return privateResourceName
			End Get
			Set(ByVal value As String)
				privateResourceName = value
			End Set
		End Property
	End Class
End Namespace
